contracts/lido_terra_rewards_dispatcher/src/contract.rs

```diff
@@ -58,6 +59,38 @@ pub fn execute(
     info: MessageInfo,
     msg: ExecuteMsg,
 ) -> StdResult<Response<TerraMsgWrapper>> {
+    if let ExecuteMsg::UpdateConfig {
+        owner,
+        hub_contract,
+        bluna_reward_contract,
+        stluna_reward_denom,
+        bluna_reward_denom,
+        lido_fee_address,
+        lido_fee_rate,
+    } = msg
+    {
+        return execute_update_config(
+            deps,
+            env,
+            info,
+            owner,
+            hub_contract,
+            bluna_reward_contract,
+            stluna_reward_denom,
+            bluna_reward_denom,
+            lido_fee_address,
+            lido_fee_rate,
+        );
+    }
+
+    let config: Config = CONFIG.load(deps.storage)?;
```

Comment:
What about to pass the config to execute_ methods, to not read it twice?

Reply:
Unnecessary reads are removed